pkg/crc/preflight/preflight_checks_windows.go

```diff
@@ -79,7 +93,11 @@ func checkIfUserPartOfHyperVAdmins() (bool, error) {
 	checkIfMemberOfHyperVAdmins :=
 		`$sid = New-Object System.Security.Principal.SecurityIdentifier("S-1-5-32-578")
 	@([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole($sid)`
-	stdOut, _, _ := powershell.Execute(checkIfMemberOfHyperVAdmins)
+	stdOut, _, err := powershell.Execute(checkIfMemberOfHyperVAdmins)
+	if err != nil {
+		logging.Debug(err.Error())
+		return false, errors.New("Failed checking if user is part of hyperv admins group")
```

Comment:
Did you check preflight on Windows after these changes? I wonder if we get an error when user is not a member of the admin group, or just regular output from the command?